modules/op-customizing-pipelines-as-code-configuration.adoc

```diff
@@ -0,0 +1,30 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="customizing-pipelines-as-code-configuration_{context}"]
+= Customizing Pipelines as Code configuration 
+
+[role="_abstract"]
+To customize Pipelines as Code, cluster administrators can configure the following parameters using the `pipelines-as-code` config map in the `pipelines-as-code` namespace:
+
+.Customizing Pipelines as Code configuration
+[options="header"]
+|===
+
+| Parameter | Description | Default
+
+| `application-name` | The name of the application. For example, the name displayed in the GitHub Checks labels. | `"Pipelines as Code CI"` 
+
+| `max-keep-days` | The number of the days for which the executed pipeline runs are kept in the `pipelines-as-code namespace`. 
+
+Note that this configmap setting does not affect the cleanups of an user’s pipeline runs, which are controlled by the annotations on the pipeline run definition in the user’s GitHub repository. |
```

Comment:
```suggestion
Note that this configmap setting does not affect the cleanups of a user’s pipeline runs, which are controlled by the annotations on the pipeline run definition in the user’s GitHub repository. |
```
- is `cleanups` the correct technical term here? Is it referring to the name of a specific feature or function? If not, then the sentence should be re-worded to say something like `does not affect how a user's pipeline runs are cleaned up`